standard/expressions.md

```diff
@@ -971,6 +971,14 @@ For a function member that includes a parameter array, if the function member is
   - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
   - for a `ref` or `out` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
 
+Additional rules determine whether a method is applicable or not based on the context of the expression:
+
+- An instance method is not applicable when the invocation has no explicit receiver and `this` cannot be used explicitly (such as in static methods, field initializers and constructor initializers).
+- A static method is not applicable when the invocation has an explicit receiver which is not a type.
+- When the receiver could be either an instance or a type as described in [§11.7.6.2](expressions.md#11762-identical-simple-names-and-type-names), both instance and static methods are applicable.
```

Comment:
Let's try to go back to the style of text we deleted - but the "this" aspect was a problem. (We could come up with an example of static+instance overloads; the instance one should be removed earlier.)

Reply:
Can refer to "where a this_access is permitted" and refer to https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#11712-this-access